test/logger/logs.go

```diff
@@ -1,13 +1,15 @@
+// Copyright Authors of Cilium
+// SPDX-License-Identifier: Apache-2.0
+
 package logger
 
 import (
 	"bytes"
 	"fmt"
 	"os"
 
-	"github.com/sirupsen/logrus"
-
 	ginkgo "github.com/cilium/cilium/test/ginkgo-ext"
+	"github.com/sirupsen/logrus"
```

Comment:
Imports are typically set into three categories separated by whitespace: First, standard library imports, second external imports from other projects, and third imports within the codebase. As such, the linter is complaining. Please revert this hunk here.